### PR TITLE
Expose reverse tunnel address to web ui

### PR DESCRIPTION
**Pull request by mcbattirola**
## Purpose

The ultimate purpose is to show the reverse tunnel address on `Help & Support` screen on the web UI.

On cloud instances, we use a custom reverse tunnel port for each tenant. In the UI, there is no way to know your reverse tunnel address, which is necessary when adding a trusted cluster.

___

This PR adds the reverse tunnel address to the `GRV_CONFIG` object that is injected into the web UI. I added it behind a feature flag to only show on cloud only, should we expose it to everyone?

Note that this field is already publicly exposed via `GET [proxy server]/webapi/ping`.

Here's the PR on `webapps` to show the field on UI: gravitational/webapps#580

Part of gravitational/cloud#950

### CODE REVIEW

File: lib/web/ui/webconfig.go [hunk 1]
```diff
@@ -45,6 +45,8 @@ type WebConfig struct {
 	ProxyClusterName string `json:"proxyCluster,omitempty"`
 	// IsCloud is a flag that determines if cloud features are enabled.
 	IsCloud bool `json:"isCloud,omitempty"`
+	// TunnelPublicAddress is the public ssh tunnel address
+	TunnelPublicAddress string `json:"tunnelPublicAddress,omitempty"`
```
Comment (russjones) [inline lib/web/ui/webconfig.go:49]: @kimlisa What's our convention, camel case or underscores?

**Comment by russjones** [comment]
@smallinsky @jakule Can you review?

File: lib/web/ui/webconfig.go (hunk 1 above)
Comment (kimlisa) [reply lib/web/ui/webconfig.go:49]: @russjones camel case (as it is)

Comment (russjones) [reply lib/web/ui/webconfig.go:49]: Okay, thank you!

**Review by smallinsky — APPROVED** [review]
LGTM

File: lib/web/apiserver.go [hunk 2]
```diff
@@ -894,10 +894,22 @@ func (h *Handler) getWebConfig(w http.ResponseWriter, r *http.Request, p httprou
 		PreferredLocalMFA: cap.GetPreferredLocalMFA(),
 	}
 
+	// get tunnel address to display on cloud instances
+	tunnelPublicAddr := ""
+	if h.ClusterFeatures.GetCloud() {
+		proxyConfig, err := h.cfg.ProxySettings.GetProxySettings(r.Context())
+		if err != nil {
+			h.log.WithError(err).Error("Cannot retrieve ProxySettings.")
```
Comment (jakule) [inline lib/web/apiserver.go:902]: `"Cannot retrieve ProxySettings."` is just logged (not returned as function error) in this case. Someone reading logs probably won't know what is going on. As this is not "fatal" error can we add something like `tunnel address won't be set in Web UI` to make it more clear and consider making this a warning?

**Review by jakule — APPROVED** [review]
One comment, looks good.

File: lib/web/apiserver.go (hunk 2 above)
Comment (mcbattirola) [reply lib/web/apiserver.go]: Thanks @jakule , this makes sense
I've changed to a to a warning and added `tunnel address won't be set in Web UI`

**Pull request merged**